backend/local/local.go

```diff
@@ -143,7 +147,16 @@ func (lb *localBlob) Finalize(t backend.Type, name string) error {
 		return err
 	}
 
-	return os.Chmod(f, fi.Mode()&os.FileMode(^uint32(0222)))
+	// set file to readonly, except on Windows,
+	// otherwise deletion will fail.
+	if runtime.GOOS != "windows" {
+		err = os.Chmod(f, fi.Mode()&os.FileMode(^uint32(0222)))
```

Comment:
I don't like that this is evaluated every time the code is called. What about adding a new function `setNewFileMode()` that does the `Chmod()` on Unix and nothing on Windows?